engine/src/flutter/lib/ui/window.dart

```diff
@@ -968,6 +969,20 @@ class AccessibilityFeatures {
   /// Only supported on iOS.
   bool get onOffSwitchLabels => _kOnOffSwitchLabelsIndex & _index != 0;
 
+  /// Whether accessibility announcements (like [SemanticsService.announce])
+  /// are allowed on the current platform.
```

Comment:
```suggestion
  /// are encouraged on the current platform.
```
right now announce is discouraged but possible on android 36.

Reply:
I think the accessibility team discourages using `announcements` in general. Should we keep it as `allowed` in that case?

Reply:
The android accessibility team discourages announcements other accessability teams have the api and have no issue.  Think about this documentation from the callers perspective. Is this value about if you can use announcements? I think not because technically they can work until it is just discourage because they want to turn off the api. Is this value about if you should or if it is encouraged or if it is some other word? But make the value have predictable meaning to a new situation.

Reply:
Ah, sorry I meant that the Flutter accessibility team also discourages announcements too. @chunhtai for more input on announce usage in Flutter. I'm thinking that supportsAnnounce would be the best because if Android ever removes announce support, then supportsAnnounce can still just be false and developer's can continue to rely on their branched logic.

Reply:
Okay gotcha. I'm leaning towards using `supports` in the docs here then. Also see similar comment here for reasoning: https://github.com/flutter/flutter/pull/168992#discussion_r2146068546